etc/kernel-launchers/R/scripts/gateway_listener.py

```diff
@@ -95,9 +95,7 @@ def gateway_listener(sock, parent_pid):
                 os.kill(int(parent_pid), signum)
             elif request.get('shutdown'):
                 shutdown = bool(request.get('shutdown'))
-                if shutdown:
-                    print("Listener received shutdown request, terminating parent (pid: {}).".format(parent_pid))
```

Comment:
Do we need to actually keep the log message around?

Reply:
I don't believe so.  This behavior is consistent with the python launcher.  However, in checking the Scala launcher just now, it does produce the following: `Stopping gateway listener.`

Since the launcher is going to terminate, I think its fine to not have a message here.  If anything, it would probably be appropriate to log as the last line in the main launcher script.